### PR TITLE
modify descriptions of column default value 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: data-type-default-values.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ aliases: ['/docs-cn/dev/data-type-default-values/','/docs-cn/dev/reference/sql/d
 
 # 数据类型的默认值
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:10]:
```suggestion
- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
```

File: data-type-default-values.md [hunk 2]
```diff
@@ -5,9 +5,11 @@ aliases: ['/docs-cn/dev/data-type-default-values/','/docs-cn/dev/reference/sql/d
 
 # 数据类型的默认值
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
 
-从 v8.0.0 开始，[`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 可以设置[表达式默认值](#表达式默认值)。
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值。
```
Comment (zimulala) [inline data-type-default-values.md:11]: 这个是否考虑放到 line48 后面。本身这里是数据类型默认值，不是表达式默认值。"NEXT VALUE FOR" 感觉是表达式默认值。时间类型放在这里我认为是一开始有这个文档的时候是参考当时的 mysql 文档。

Comment (qiancai) [reply data-type-default-values.md:11]: line48 后面在描述 v8.0 新增的实验特性，但next val 作为列的默认值不是实验特性，所以为了避免混淆，可以保留在这里？

**Review by Frank945946 — APPROVED** [review]

File: data-type-default-values.md (hunk 2 above)
Comment (Frank945946) [reply data-type-default-values.md:14]: 48 行下面 要不重点突出一下 8.0 开始支持的表达式？ @zimulala

**Review by zimulala — APPROVED** [review]

File: data-type-default-values.md (hunk 2 above)
Comment (zimulala) [reply data-type-default-values.md:14]: 可以的

File: data-type-default-values.md [hunk 3]
```diff
@@ -34,32 +38,31 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
-TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。从 v8.0.0 开始，TiDB 额外支持 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
-
-```sql
-CREATE TABLE t2 (b BLOB DEFAULT (RAND()));
-```
-
-TiDB 支持以下表达式：
-
-* [`RAND()`](/functions-and-operators/numeric-functions-and-operators.md)
-* [`UUID()`](/functions-and-operators/miscellaneous-functions.md)
```
Comment (zimulala) [inline data-type-default-values.md:46]: 这两个表达式删了？

File: data-type-default-values.md [hunk 4]
```diff
@@ -34,32 +38,31 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
-从 v8.0.0 开始，在 `DEFAULT` 子句中可以使用以下表达式来设置默认值。
-
-* `UPPER(SUBSTRING_INDEX(USER(), '@', 1))`
-
-* `REPLACE(UPPER(UUID()), '-', '')`
-
-* `DATE_FORMAT` 相关表达式，具体格式如下：
+TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。而在 v8.0.0 版本，新增以下功能。
+
+1. 在 `DEFAULT` 子句中可以使用以下新增的表达式来设置默认值：
```
Comment (qiancai) [inline data-type-default-values.md:43]:
```suggestion
TiDB 从 v8.0.0 开始支持在 `DEFAULT` 子句中指定以下表达式作为字段的默认值。

```

File: data-type-default-values.md [hunk 5]
```diff
@@ -34,32 +38,31 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
+   * `DATE_FORMAT` 相关表达式，具体格式如下：
+   
+       * `DATE_FORMAT(NOW(), '%Y-%m')`
+       * `DATE_FORMAT(NOW(), '%Y-%m-%d')`
+       * `DATE_FORMAT(NOW(), '%Y-%m-%d %H.%i.%s')`
+       * `DATE_FORMAT(NOW(), '%Y-%m-%d %H:%i:%s')`
+   
+   * `STR_TO_DATE('1980-01-01', '%Y-%m-%d')`
+     
+2. TiDB 额外支持 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```
Comment (qiancai) [inline data-type-default-values.md:59]:
```suggestion
此外，从 v8.0.0 开始，TiDB 额外支持 `BLOB`、`TEXT` 以及 `JSON` 数据类型分配默认值，但是默认值仅支持通过表达式来设置。以下是 `BLOB` 的示例：
```

File: data-type-default-values.md [hunk 6]
```diff
@@ -6,9 +6,13 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 # 数据类型的默认值
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:14]:
```suggestion
- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/)) 函数
- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数
- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数
```

File: data-type-default-values.md [hunk 7]
```diff
@@ -34,32 +38,26 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
-    * `DATE_FORMAT(NOW(), '%Y-%m-%d %H.%i.%s')`
-    * `DATE_FORMAT(NOW(), '%Y-%m-%d %H:%i:%s')`
-
-* `STR_TO_DATE('1980-01-01', '%Y-%m-%d')`
+TiDB 参考了该功能，支持在 `DEFAULT` 子句中指定部分表达式作为字段的默认值。而在 v8.0.0 版本，新增以下功能。
+
+1. 在 `DEFAULT` 子句中可以使用以下新增的表达式来设置默认值：
+   * [`RAND()`](/functions-and-operators/numeric-functions-and-operators.md)
+   * [`UUID()`](/functions-and-operators/miscellaneous-functions.md)
+   * [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md)
```
Comment (qiancai) [inline data-type-default-values.md:46]:
```suggestion
```

File: data-type-default-values.md [hunk 8]
```diff
@@ -6,9 +6,13 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值。
+- 从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```
Comment (qiancai) [inline data-type-default-values.md:15]:
```suggestion
- 从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
- 
```

File: data-type-default-values.md [hunk 9]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/)) 函数
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数
+- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数
+- 从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```
Comment (qiancai) [inline data-type-default-values.md:17]:
```suggestion
- 从 v8.0.0 开始，在 `DEFAULT` 子句中新增支持指定[部分表达式](#表达式默认值)作为字段的默认值。此外，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```

File: data-type-default-values.md [hunk 10]
```diff
@@ -11,10 +11,10 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
-- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数
-- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数
-- 从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/) 函数生成随机浮点值作为列的默认值。
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
+- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数将 UUID 转换为二进制格式后作为列的默认值。
+- 从 v8.0.0 开始，在 `DEFAULT` 子句中新增支持指定[部分表达式](#表达式默认值)作为字段的默认值。此外，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```
Comment (Frank945946) [inline data-type-default-values.md:17]: “在 `DEFAULT` 子句中新增支持指定[部分表达式](#表达式默认值)作为字段的默认值。此外，” 这句话有点冗余，可以删除掉。因为后半句已经提到了给这三种数据类型通过表达式设置默认值。

**Review by Frank945946 — APPROVED** [review]

File: data-type-default-values.md [hunk 11]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/) 函数生成随机浮点值作为列的默认值。
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
+- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数将 UUID 转换为二进制格式后作为列的默认值。
+- 从 v8.0.0 开始，在 `DEFAULT` 子句中新增支持指定[部分表达式](#表达式默认值)作为字段的默认值。此外，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```
Comment (Frank945946) [inline data-type-default-values.md:17]:
```suggestion
- 从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
```

**Review by Frank945946 — APPROVED** [review]
Rest LGTM

File: data-type-default-values.md (hunk 3 above)
Comment (qiancai) [reply data-type-default-values.md:46]: moved to L15

File: data-type-default-values.md [hunk 12]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/) 函数生成随机浮点值作为列的默认值。
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:15]:
```suggestion
- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/miscellaneous-functions.md) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
```

File: data-type-default-values.md [hunk 13]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 # 数据类型的默认值
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/[numeric-functions-and-operators.md](http://numeric-functions-and-operators.md/) 函数生成随机浮点值作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:14]:
```suggestion
- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/numeric-functions-and-operators.md) 函数生成随机浮点值作为列的默认值。
```

File: data-type-default-values.md [hunk 14]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/[miscellaneous-functions.md](http://miscellaneous-functions.md/)) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
+- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数将 UUID 转换为二进制格式后作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:16]:
```suggestion
- 对于二进制类型，可以使用 [`UUID_TO_BIN()`](/functions-and-operators/miscellaneous-functions.md) 函数将 UUID 转换为二进制格式后作为列的默认值。
```

File: data-type-default-values.md [hunk 15]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/numeric-functions-and-operators.md) 函数生成随机浮点值作为列的默认值。
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/miscellaneous-functions.md) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md)作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:15]:
```suggestion
- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/miscellaneous-functions.md) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md) 作为列的默认值。
```

**Review by hfxsd — APPROVED** [review]

File: data-type-default-values.md [hunk 16]
```diff
@@ -6,9 +6,15 @@ summary: 数据类型的默认值描述了列的默认值设置规则。默认
...
 
-在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。这个默认值必须是常量，不可以是一个函数或者是表达式。但是对于时间类型，可以例外的使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 的默认值。
+在一个数据类型描述中的 `DEFAULT value` 段描述了一个列的默认值。
 
-所有数据类型都可以设置默认值。从 v8.0.0 开始，新增支持 [`BLOB`](/data-type-string.md#blob-类型)、[`TEXT`](/data-type-string.md#text-类型) 以及 [`JSON`](/data-type-json.md#json-类型) 这三种数据类型设置默认值，但仅支持使用表达式设置[默认值](#表达式默认值)。
+所有数据类型都可以设置默认值。这个默认值通常情况下必须是常量，不可以是一个函数或者是表达式，但也存在以下例外情况：
+
+- 对于时间类型，可以使用 `NOW`、`CURRENT_TIMESTAMP`、`LOCALTIME`、`LOCALTIMESTAMP` 等函数作为 `DATETIME` 或者 `TIMESTAMP` 列的默认值。
+- 对于整数类型，可以使用 `NEXT VALUE FOR` 函数将序列的下一个值作为列的默认值，使用 [`RAND()`](/functions-and-operators/numeric-functions-and-operators.md) 函数生成随机浮点值作为列的默认值。
+- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/miscellaneous-functions.md) 函数生成通用[唯一标识符 (UUID)](/best-practices/uuid.md) 作为列的默认值。
```
Comment (qiancai) [inline data-type-default-values.md:15]:
```suggestion
- 对于字符串类型，可以使用 [`UUID()`](/functions-and-operators/miscellaneous-functions.md) 函数生成[通用唯一标识符 (UUID)](/best-practices/uuid.md) 作为列的默认值。
```

**Pull request merged**